core/modules/update/update.theme.inc

```diff
@@ -272,7 +273,7 @@ function theme_update_report($variables) {
         $sub_section .= "</div>\n";  // .subthemes div
       }
 
-      $info_section = '<div class="info">' . $extra_section . $includes_section . $base_section . $sub_section . '</div>';
+      $info_section = '<div class="project-info">' . $extra_section . $includes_section . $base_section . $sub_section . '</div>';
```

Comment:
Changes needed in `core/modules/update/css/update.css`

- https://github.com/backdrop/backdrop/blob/3d5074a261c5af713286614d1aa2048919e69b00/core/modules/update/css/update.css#L36 
- https://github.com/backdrop/backdrop/blob/3d5074a261c5af713286614d1aa2048919e69b00/core/modules/update/css/update.css#L127-L134

Reply:
Good catch @opi 👍. Updated PR.